src/code/components/clutch-view.js

```diff
@@ -0,0 +1,84 @@
+import React, {PropTypes} from 'react';
```

Comment:
At the moment, a `clutch-view` is basically a `stable-view`, but with pages and interactivity. In the future, it should add more functionality, like setting grid parameters, so I broke it into it's own component here.